Pokitto/POKITTO_LIBS/Tilemap/Tilemap.hpp

```diff
@@ -79,8 +79,11 @@ class Tilemap {
         for( uint32_t ty = 0; ty<height; ty++ ){
             cx = x;
 
-            // Note: Each 8-bit map item contains 2 tiles.
             uint32_t h = 0;
+
+            #if MAX_TILE_COUNT == 16
```

Comment:
Can you use a class template instead?
`template<int MAX_TILE_COUNT = 16> class Tilemap ...`
Then just use regular `if` instead of `#if`.